packages/web/src/scss/tools/_form-fields.scss

```diff
@@ -94,10 +94,24 @@
     }
 }
 
-@mixin box-field-variants($field-name) {
+@mixin input-field-validation-states($field-name, $shadow-x: null, $shadow-y: null) {
```

Comment:
Personally, I just find `validation-state' too common. That's why I left at least the `input-field-` there, because they are styles for all `<input>` elements.